src/rapidfuzz/process.pyi

```diff
@@ -1,35 +1,36 @@
-from typing import Any, Mapping, Tuple, Callable, Hashable, Sequence, Iterable, Optional, Union, overload, TypeVar, List, Generator
+from typing import Any, Collection, Mapping, Tuple, Callable, Hashable, Sequence, Optional, Union, overload, TypeVar, List, Generator
 from rapidfuzz.fuzz import WRatio
 
 _StringType = Sequence[Hashable]
+_AnyStringType = TypeVar("_AnyStringType", bound=_StringType)
 S1 = TypeVar("S1")
 S2 = TypeVar("S2")
 ResultType = TypeVar("ResultType", int, float)
 
 @overload
 def extractOne(
     query: _StringType,
-    choices: Iterable[_StringType], *,
+    choices: Collection[_AnyStringType], *,
```

Comment:
extractOne and extract_iter do not use the container length. So it should work with Iterables and not only Collections.

Reply:
Thanks, fixed.